pkg/localstorev2/storer.go

```diff
@@ -212,9 +212,15 @@ func initDiskRepository(basePath string, opts *Options) (storage.Repository, io.
 	}
 
 	txStore := leveldbstore.NewTxStore(store)
+
+	recoveryCloser, err := sharkyRecovery(sharkyBasePath, txStore, opts)
```

Comment:
This should be done before we start the Sharky instance as they will be operating on the same set of files underneath. This will be wrong.